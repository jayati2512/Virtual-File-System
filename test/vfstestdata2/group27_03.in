createvfs 30
createvfs gh/sh 50
createvfs thenameofthisvfsissolongthatyoucannotcreateit 30
mountvfs thenameofthisvfsissolongthatyoucannotcreateit
createvfs group27 40
makedir / home
mountvfs group27
makedir / home
makedir etc
mountvfs group27
makedir / tmp
makedir / home
makedir /home/ abc
makedir /home/ abc
unmountvfs group27
unmountvfs group27
mountvfs group27
deletedir /home/abc/
makedir /home/ def
makedir / hft
movedir /hft/ /home/def/
listdir / 1 /tmp/output_listdir.txt
unmountvfs group27
