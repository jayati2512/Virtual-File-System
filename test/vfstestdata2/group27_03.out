createvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
createvfs_FAILURE INVALID_CHARACTER_IN_NAME
createvfs_FAILURE VFS_LABEL_TOO_LARGE
mountvfs_FAILURE DATA_FILE_NOT_FOUND
createvfs_SUCCESS
makedir_FAILURE VFS_NOT_MOUNTED
mountvfs_SUCCESS
makedir_SUCCESS
makedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
mountvfs_FAILURE VFS_ALREADY_MOUNTED
makedir_SUCCESS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
makedir_SUCCESS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
unmountvfs_SUCCESS
unmountvfs_VFS_NOT_MOUNTED
mountvfs_SUCCESS
deletedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
movedir_SUCCESS
listdir_SUCCESS
unmountvfs_SUCCESS
